honeybee_schema/radiance/modifierset.py

```diff
@@ -0,0 +1,327 @@
+
+    type: constr(regex='^ShadeModifierSetAbridged$') = 'ShadeModifierSetAbridged'
+
+
+class ApertureModifierSetAbridged(NoExtraBaseModel):
+    """Abridged set containing radiance modifiers needed for a model's Apertures."""
+
+    type: constr(regex='^ApertureModifierSetAbridged$') = 'ApertureModifierSetAbridged'
+
+    window_modifier: str = Field(
+            default=None,
+            description='Identifier of modifier object for apertures with an Outdoors '
+                        'boundary condition, False is_operable property, '
+                        'and Wall parent Face.'
+        )
+
+    interior_modifier: str = Field(
+            default=None,
+            description='Identifier of modifier object for apertures with a Surface '
+                        'boundary condition.'
+        )
+
+    skylight_modifier: str = Field(
+            default=None,
+            description='Identifier of modifier object for apertures with an Outdoors '
+                        'boundary condition, False is_operable property, and a '
+                        'RoofCeiling or Floor face type for their parent face.'
+        )
+
+    operable_modifier: str = Field(
+            default=None,
+            description='Identifier of modifier object for apertures with an Outdoors '
+                        'boundary condition and a True is_operable property.'
+        )
+
+
+class DoorModifierSetAbridged(BaseModifierSetAbridged):
+    """Abridged set containing radiance modifiers needed for a model's Doors."""
+
+    type: constr(regex='^DoorModifierSetAbridged$') = 'DoorModifierSetAbridged'
+
+    exterior_modifier: str = Field(
+            default=None,
+            description='Identifier of modifier object for doors with an Outdoors '
+                        'boundary condition.'
+        )
+
+    interior_glass_modifier: str = Field(
+            default=None,
+            description='Identifier of modifier object for glass with a Surface '
+                        'boundary condition.'
+        )
+
+    exterior_glass_modifier: str = Field(
+            default=None,
+            description='Identifier of modifier object for glass with an Outdoors '
+                        'boundary condition.'
+        )
+
+    overhead_modifier: str = Field(
+            default=None,
+            description='Identifier of window modifier object for doors with an '
+                        'Outdoors boundary condition and a RoofCeiling or Floor '
+                        'face type for their parent face.'
+        )
+
+
+class ModifierSetAbridged(IDdRadianceBaseModel):
+    """Abridged set containing all modifiers needed to create a radiance model."""
+
+    type: constr(regex='^ModifierSetAbridged$') = 'ModifierSetAbridged'
+
+    wall_set: WallModifierSetAbridged = Field(
+        default=None,
+        description='Identifer for optional WallModifierSet object for this '
+                    'ModifierSet (default: None).'
```

Comment:
All of these descriptions for the sub-sets are incorrect at the moment. This is not an identifer. It's a `WallModifierSetAbridged` object.